clients/python/game/models.py

```diff
@@ -1,18 +1,32 @@
+class UnknownPlayerIdException(Exception):
+    def __init__(self, player_id):
+        super().__init__("Unknown Player ID %d" % player_id)
+
+
 class Game:
     RankedGameId = 0
 
-    def __init__(self, id_, tick, players, resources, map_, viruses):
+    def __init__(self, id_, tick, player_id, players,
+                 resources, map_, viruses):
         self.id = id_
         self.tick = tick
         self.players = players
         self.resources = resources
         self.map = map_
         self.viruses = viruses
 
-    def parse(obj):
+        with_my_id = [player for player in players if player.id == player_id]
```

Comment:
Muuuuch better with `next`, thanks! Do you like how it looks now? If so, please merge!